camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeper.java

```diff
@@ -222,16 +232,11 @@ public void run() throws Exception
       {
         System.err.println("unable to process " + topicName + " skipping...");
         e.printStackTrace();
-        executorService.shutdown();
-        throw e;
```

Comment:
I'd guess all kinds of exception from DivisionByZero to out of memory exception - you will have to contact Linkedin for more exact answer.